static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -245,3 +245,321 @@
+        }
+        .meta__numbers {
+            color: $neutral-3;
+        }
+        .content__dateline, .sharecount__heading {
+            color: $neutral-2;
+        }
+        .byline a, .sharecount__value {
+            color: $neutral-3;
+        }
+    }
+
+    .content__wrapper--headline {
+        background-color: rgba(0, 0, 0, .6);
+        background: linear-gradient(rgba(0, 0, 0, .5), $multimedia-support-5);
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: 0;
+    }
+
+    .content__headline {
+        &:before {
+            content: none;
+        }
+    }
+
+    .inline-camera svg {
+        width: 32px;
+        height: auto;
+    }
+
+    .content__standfirst {
+        color: $neutral-5;
+    }
+
+    .content__meta-container {
+        border: 0;
+        position: relative;
+        width: auto;
+        margin-bottom: 0;
+    }
+
+    .meta__social {
+        border-top: 1px dotted $neutral-2;
+        border-bottom: 1px dotted $neutral-2;
+    }
+
+    .meta__break {
+        border-top: 1px dotted $neutral-2;
+        display: block;
+        width: 10%;
+    }
+
+    .meta__numbers {
+        border: 0;
+        position: absolute;
+        right: 0;
+        top: 0;
+
+        .sharecount__heading {
+            padding-left: 0;
+        }
+    }
+
+    .content__series-label {
+        padding-top: 0;
+        margin-left: 1rem;
+    }
+
+    .content__series-label, .content__section-label {
+        display: inline-block;
+    }
+
+    .content__labels {
+        border: 0;
+        margin-bottom: 0;
+        width: auto;
+        float: none;
+    }
+
+    .block-share--gallery {
+        position: relative;
+        .block-share__item--facebook,
+        .block-share__item--twitter,
+        .block-share__item--pinterest {
+            background-color: rgba(0, 0, 0, 0);
+            border: 1px solid $neutral-2;
+
+            &:hover {
+                background-color: rgba(0, 0, 0, 0);
+            }
+
+            &.inline-icon, .inline-icon {
+                fill: $neutral-3;
+            }
+        }
+    }
+
+    .social-icon, .block-share__item {
+        background-color: rgba(0, 0, 0, 0);
+        border: 1px solid $neutral-2;
+
+        &:hover {
+            background-color: rgba(0, 0, 0, 0);
+        }
+
+        &.inline-icon, .inline-icon {
+            fill: $neutral-3
+        }
+    }
+
+    .gallery {
+        list-style: none;
+        margin: 0;
+    }
+
+    .gallery__item {
+        figure {
+            display: flex;
+            flex-direction: column-reverse;
+        }
+
+        // On mobile the images should be 100% of the width of the screen
+        @include mq($until: phablet) {
+            margin-left: -$gs-gutter;
+            margin-right: -$gs-gutter;
+        }
+        @include mq($until: mobileLandscape) {
+            margin-left: -$gs-gutter / 2;
+            margin-right: -$gs-gutter / 2;
+        }
+
+        @include mq(tablet) {
```

Comment:
This media query has empty line before, one above doesn't. Again I'd opt for spacing them out but it's personal preference.